docs/dev/cut-a-new-release.md

```diff
@@ -1,38 +1,58 @@
-        3. Optionally, go over the commits between the past release and this one and see if anything is worth including.
-        4. Make sure the changelog follows the format of the other changelog entries.
-            - Note that we try to provide PR or issue numbers (and links) for each change. Please add these if they are missing.
-
-    2. Add a new `# Unreleased Changes` on top
-
-2. Bump the versions
-    * Rust crates: Bump `version` in [`glean-core/Cargo.toml`](https://github.com/mozilla/glean/blob/master/glean-core/Cargo.toml) and [`glean-core/ffi/Cargo.toml`](https://github.com/mozilla/glean/blob/master/glean-core/ffi/Cargo.toml).  **NOTE:** There are two versions to update in `glean-core/ffi/Cargo.toml`.
-    * Kotlin package: Bump `libraryVersion` in the top-level [.buildconfig.yml](https://github.com/mozilla/glean/blob/master/.buildconfig.yml) file.
-    * Gradle plugin: Bump `project.ext.gleanVersion` in [`GleanGradlePlugin.groovy`](https://github.com/mozilla/glean/blob/master/gradle-plugin/src/main/groovy/mozilla/telemetry/glean-gradle-plugin/GleanGradlePlugin.groovy).
-    * Be sure you're following semver, and if in doubt, ask.
-    * Run `make build` to make sure the `Cargo.lock` file is updated.
-3. Land the commits that perform the steps above. This takes a PR, typically, because of branch protection on master.
+    1. Add any missing important changes under the `Unreleased changes` headline
+2. Run `bin/prepare-release.sh <new version>`
+    1. The new version should be the next patch, minor or major version of what is currently released.
+    2. Let it create a commit for you.
+3. Land the new commit that perform the steps above. This takes a PR, typically, because of branch protection on master.
 4. Cut the actual release.
     1. Click "Releases", and then "Draft a New Release" in the github UI.
-    2. Enter `v<myversion>` as the tag. In the example above it would be `v0.1.0`. It's important this is the same as the tags you put in the links in the changelog.
+    2. Enter `v<myversion>` as the tag. It's important this is the same as the tags you put in the links in the changelog.
     3. Under the description, paste the contents of the release notes from `CHANGELOG.md`.
     4. Note that the release is not available until the CI build completes for that tag.
         - You can check [on CircleCI for the running build](https://circleci.com/gh/mozilla/glean).
-5. After the release, bump the versions to the next pre-release:
-    * Rust crates: Use the next major version and append `-alpha.1`
-    * Kotlin package: Use the next major version and append `-SNAPSHOT`
+    5. Release the Rust crates:
+
+       ```
+       cd glean-core
+       cargo publish --verbose
+       cd ffi
+       cargo publish --verbose
+       ```
+
+> **Important:**
+> The release process produces release artifacts for the Kotlin library.
```

Comment:
It also publishes wheels to PyPI (as of yesterday).  There is a follow-on [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1598021) to also push those wheels to github releases, just so everything will also be in one place.